src/pages/profile/components/profile-form/index.tsx

```diff
@@ -0,0 +1,280 @@
+import {
+	Box,
+	Button,
+	FormControl,
+	FormHelperText,
+	InputLabel,
+	MenuItem,
+	Select as MuiSelect,
+	Typography,
+} from "@mui/material";
+import React from "react";
+import { useTranslation } from "react-i18next";
+import {
+	CountryData,
+	PhoneInput,
+	defaultCountries,
+	parseCountry,
+} from "react-international-phone";
+import "react-international-phone/style.css";
+
+import {
+	BaseButton,
+	CustomFormGroup,
+	ImagePreview,
+} from "~/components/index.ts";
+import { fontSizes } from "~/libs/constants/fonts.ts";
+import {
+	clothingSizes,
+	jeansSizes,
+	shoeSizes,
+} from "~/pages/profile-board/constants.ts/size.ts";
+import theme from "~/theme.ts";
+
+import { useProfileForm } from "../../hooks/use-profile-form.ts";
+import { ProfileAddress, ProfileCard } from "../index.ts";
+import {
+	StyledFormLabel,
+	StyledPhoneCountryInput,
+	StyledPhoneInput,
+	VisuallyHiddenInput,
+} from "./styles.ts";
+
+const ProfileForm: React.FC = () => {
+	const { t } = useTranslation();
+	const {
+		control,
+		errors,
+		handleChangePhone,
+		handleClothingSizeChange,
+		handleFileChange,
+		handleFormSubmit,
+		handleJeansSizeChange,
+		handleShoeSizeChange,
+		phone,
+		selectedClothingSize,
+		selectedFile,
+		selectedJeansSize,
+		selectedShoeSize,
+		serverError,
+		user,
+	} = useProfileForm();
+
+	const requiredCodes = ["ua", "ca"];
+
+	const countries = defaultCountries.filter((country: CountryData) => {
+		const { iso2 } = parseCountry(country);
+
+		return requiredCodes.includes(iso2);
+	});
+
+	return (
+		<Box
+			sx={{
+				display: "flex",
+				flexDirection: "column",
+				gap: "24px",
+				padding: "24px",
+			}}
+			width="80%"
```

Comment:
Component too big So move some mui component with sx props to styles.ts and use it like Styled Profile Box or something like this. You can do it for several component in this form.
P.S. No one told me about it and i maked this mistake in Onboarding, but now i tell it to you(